tac/project-updates/2025/2025-annual-Hyperledger-Indy.md

```diff
@@ -0,0 +1,138 @@
+
+## Maintainer Diversity
+
+There are 23 current maintainers on 18 Indy GitHub Teams representing at least 13 organizations. This represents a decrease of 12 total maintainers but only 3 fewer organizations compared to last year. The maintainers and teams can be seen in the Hyperledger [Access Control YAML file]https://github.com/hyperledger/governance/blob/main/access-control.yaml). The Indy maintainers file ([example](https://github.com/hyperledger/indy/blob/main/MAINTAINERS.md)) links to the Access Control YAML as the "source of truth."
+
+## Project Adoption
+
+Several private Indy networks were created during the past year, with some being preparatory for production usage of Hyperledger Indy. For one example, a travel technology company built both a test and a demo network during 2024 and plans to build a production Indy network during 2025.
+
+The [Use Case Tracker](https://www.lfdecentralizedtrust.org/learn/use-case-tracker) shows 12 projects that are using Indy.
+
+## Goals
+
+### Performance Against Prior Goals
+
+Below is a copy of the goals from last years [Hyperledger Indy Annnual Report](https://github.com/hyperledger/toc/blob/gh-pages/project-reports/2024/2024-annual-Hyperledger-Indy.md).
+
+* Increased awareness of Indy, its usefulness, and a corresponding increase in contributors to enable it to continue to evolve.
+* Formally release the Ubuntu 20.04 release of Indy Node.
+    * Look into updating the solution to later releases.
+    * STATUS: **Completed**. The Ubuntu 20.04 official formal release of Indy node was accomplished February 2, 2024. The Ubuntu 22.04 release of Indy node had significant work performed on it during the year and will be completed during 2025. 
+* Tools and guidance in updating existing networks to the 20.04 release.
+    * STATUS: **Completed**. Documentation repo's were updated to include instructions for building networks based on the Ubuntu 20.04 codebase with the latest update being done in March, 2024.
+* Continue to support the migration of Indy clients away from the Indy SDK.
+    * Update the Indy documentation to point people to current “best practices”.
+    * Update and release the Indy VDR component as needed.
+    * STATUS: **Completed**. Indy documentation and all Discord and other community communications all point people away from the deprecated Indy SDK and towards its replacements.
+* Address the “Corporate Firewall” issue, where users behind a corporate firewall cannot access an Indy network because of the ZMQ ports being used.
+    * STATUS: **Partially Completed**. Several implementers (teams in Brazil and Canada) implemented "one-off" solutions to this using a local instance of an "indy-proxy". A Hyperledger Mentorship Program Project called "Indy Read Replica" ([repository](https://github.com/hyperledger/indy-read-replica)) implements a more general approach, allowing anyone to run a component that is configured to retrieve and server Indy transactions from multi 
```

Comment:
[nitpick] The sentence appears to be incomplete. Did you mean to continue and say "retrieve and serve Indy transactions from multiple environments"?

Reply:
Fixed